tasks/annotate-tag.js

```diff
@@ -0,0 +1,12 @@
+const shell = require('shelljs');
+const clipboardy = require('clipboardy');
+
+const [oldTag, newTag] = shell.exec('git tag', {silent: true}).tail({'-n': 2}).stdout.trim().split('\n');
+
+const compareText = `Changes: https://github.com/dideler/toggle-youtube-comments/compare/${oldTag}...${newTag}`
+
+// overwriting the lightweight tag with an annotated tag.
+shell.exec(`git tag --annotate --force ${newTag} ${newTag}^0 --message "${compareText}"`)
+
+// Copy markdown text.
```

Comment:
What do you think about wrapping these commands in named functions instead of using comments to describe the code?

Reply:
Well, I don't know until I write it. 🤔 
Once written, if named function wrap seems better, I'll change it and resubmit it!